doofinder-for-woocommerce/includes/class-landing-cache.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Doofinder\WP;
+
+use WP_REST_Response;
+use WP_REST_Request;
+use Doofinder\WP\Multilanguage\Multilanguage;
+use Doofinder\WP\Multilanguage\No_Language_Plugin;
+
+class Landing_Cache
+{
+
+    const DF_LANDING_CACHE = "df_landing_cache";
+
+
+    public static function register_endpoint()
+    {
+        register_rest_route('doofinder/v1',  '/clear_landing_cache', array(
+            'methods' => 'POST',
+            'callback' => array(Landing_Cache::class, 'doomanager_clear_cache'),
+            'permission_callback' => '__return_true'
+        ));
+    }
+
+    public static function set_cache($lang_cache, $landing_data)
+    {
+        // Cache the data for 15 minutes (900 seconds).
+        set_transient($lang_cache, $landing_data, 900);
```

Comment:
This should work flawlessly, but I have a small suggestion to make it even more readable (and with this change, the comment might be removed):

`set_transient($lang_cache, $landing_data, 15 * MINUTE_IN_SECONDS);`

See: https://codex.wordpress.org/Easier_Expression_of_Time_Constants

What do you think? :smile:

Reply:
Yes, u are right!